### DISCUSSION TITLE
Printful API integration? Is it possible?

### DISCUSSION THREAD

**Discussion by BryanSouza91** [saleor/saleor#7865] [Q&A]
I am curious about using saleor to deploy a couple of e-commerce websites and since I plan to use printful for the products I want to know how I might integrate [the printful api](https://www.printful.com/api) with saleor if at all possible? Thanks

**Comment by karolkielecki** [comment]
@BryanSouza91 Hi Bryan,
I'm not familiar with Printful, but what would you like to achieve? Would you like to manage your products on the Printful site?

**Reply by BryanSouza91** [reply]
Hello, 

Not really concerned with the management of the products, that can be done through printful's website easily. 

I was hoping to use my printful products in the saleor store. The order information has to be sent to printful for fulfillment. Printful will make items when ordered or as the store operator I can have some items manufactured and then stored in inventory at printful. Either way the order information has to be sent to them for completion. If I understand the functionality of saleor correctly I believe I could achieve this if there is a way to add additional custom functions to the backend that would send the order information to printful upon successful payment. 

Thank you

**Comment by BryanSouza91** [comment]
I missed the Extensions section of the docs for saleor. reading them now.

**Reply by karolkielecki** [reply]
Maybe creating an [app](https://docs.saleor.io/docs/3.0/developer/extending/apps/key-concepts) would work in this case. Even if the management of the products is easy through Printul, as you mentioned, then still products should be managed on the Saleor's side. I haven't seen any project powered by Saleor that the management of the products is outside of Saleor.
